llvm/test/CodeGen/PowerPC/ctrloop-cpsgn.ll

```diff
@@ -1,28 +0,0 @@
-; RUN: llc -verify-machineinstrs < %s -mcpu=ppc | FileCheck %s
```

Comment:
Lost this test?

Reply:
according to https://github.com/llvm/llvm-project/commit/1cf48ab811b341cf2ee3d7e9a9c97cf2d7b53f64
> Don't form PPC CTR-based loops around a copysignl call
copysign/copysignf never become function calls (because the SDAG expansion code
does not lower to the corresponding function call, but rather directly
implements the associated logic), but copysignl almost always is lowered into a
call to the requested libm functon (and, thus, might clobber CTR).

So it seemed like it was now irrelevant.

Reply:
Inserted a test with the baseline at start for this so you can examine the diff for it, if you want.